pkg/deploy/devconfig.go

```diff
@@ -94,7 +94,9 @@ func DevConfig(_env env.Core) (*Config, error) {
 				SubscriptionID:      _env.SubscriptionID(),
 				RPResourceGroupName: os.Getenv("USER") + "-aro-" + _env.Location(),
 				Configuration: &Configuration{
+					AzureCloudName:       &_env.Environment().ActualCloudName,
```

Comment:
```suggestion
					AzureCloudName:       to.StrPtr(&_env.Environment().ActualCloudName),
```

Reply:
`ActualCloudName` is defined as a string in `pkg/util/azureclient/environments.go`.

Reply:
using `&` is fine, I'm fine either way. `to.StrPtr` is a little more consistent